src/components/ArticleDetail/Vote/Vote.jsx

```diff
@@ -0,0 +1,44 @@
+import './Vote.css'
+import React, { useState } from 'react';
+import { updateArticleVotes } from '../../../utils/api'; 
+
+
+const Vote = ({ articleId, currentVotes }) => {
+    const [votes, setVotes] = useState(currentVotes); 
+    const [buttonClicked, setButtonClicked] = useState(false); 
+
+    const updateVotes = async (voteChange) => {
+        try {
+            setButtonClicked(true)
+            const updatedVotes = await updateArticleVotes(articleId, voteChange);
```

Comment:
to handle this, when this request is successful, set button clicked back to false again to allow another click